apps/api/src/app/widgets/usecases/get-subscriber-preference/get-subscriber-preference.usecase.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { SubscriberPreferenceRepository, NotificationTemplateRepository, NotificationTemplateEntity } from '@novu/dal';
+import { GetSubscriberPreferenceCommand } from './get-subscriber-preference.command';
+
+@Injectable()
+export class GetSubscriberPreference {
+  private notificationTemplateRepository = new NotificationTemplateRepository();
```

Comment:
why not have it in the constructor here as always and use nestjs DI (dependency injection)?

Reply:
totally right meant it